gradle.properties

```diff
@@ -1,5 +1,5 @@
 TILEDB_GIT_REPOSITORY=https://github.com/TileDB-Inc/TileDB
-TILEDB_GIT_TAG=2.1.2
+TILEDB_GIT_TAG=dev
```

Comment:
Should we wait until we release 2.2 with null support?

Reply:
Yes, I agree. I changed the tag to `dev` just for the CI. So let's wait for the 2.2 release.